README.md

```diff
@@ -32,7 +32,7 @@ Prerequisites for building and running these device plugins include:
 
 - Appropriate hardware
 - A fully configured [Kubernetes cluster]
-- A working [Go environment], of at least version v1.13.
+- A working [Go environment], of at least version v1.13 and `$GOPATH` set.
```

Comment:
nitpick: technically, I believe `go` no longer needs `GOPATH` to be set, and has a built in default - but - not having it set in the environment makes scripting just slightly trickier (you can use `$(go env GOPATH)` instead of `${GOPATH}`).
Just noting, not expecting us to fix it across the code base.

Reply:
good comment. I did not think of `$(go env GOPATH)` . I tried to minimize places to touch ;)

Reply:
good comment. I did not think of `$(go env GOPATH)` . I tried to minimize places to touch ;)

Reply:
updated.